plonky2x/core/src/frontend/hash/sha/curta/request.rs

```diff
@@ -8,16 +8,17 @@ pub enum SHARequestType {
     Variable,
 }
 
-/// A SHA request.
+/// A request for a SHA computation.
 #[derive(Debug, Clone)]
 pub enum SHARequest {
     /// A message of fixed length.
     Fixed(Vec<ByteVariable>),
-    /// A message of variable length, with the actual legnth given by the Variable.
+    /// A message of variable length, represented by a tuple `(total_message, lengh, chunk_index)`.
```

Comment:
`final_chunk_index` might be more clear here, up to you.

Reply:
`last_chunk` is used elsewhere in the codebase, is there a reason why `chunk_index` is used here?